types/karma-jasmine/index.d.ts

```diff
@@ -1,10 +1,33 @@
-// Type definitions for karma-jasmine plugin
-// Project: https://github.com/karma-runner/karma-jasmine
+// Type definitions for karma-jasmine 3.1
+// Project: https://github.com/karma-runner/karma-jasmine#readme
 // Definitions by: Michel Salib <https://github.com/michelsalib>
+//                 Piotr Błażejewicz <https://github.com/peterblazejewicz>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.8
-
+// TypeScript Version: 3.2
 /// <reference types="jasmine" />
 
-declare function fdescribe(description: string, specDefinitions: () => void): void;
-declare function fit(expectation: string, assertion: () => void): void;
+import karma = require('karma');
+
+declare module 'karma' {
+    interface ClientOptions {
```

Comment:
I'm not very familiar with module plugins - do you need to export this? I'm following:
https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-plugin-d-ts.html

Reply:
Hey, thanks for asking!
Here IMO that is not required, as I'm just augmenting existing module definition without adding new types. Trying to use export here will results in `strict-export-declare-modifiers`